docs/src/app/components/pages/components/TimePicker/ExampleComplex.jsx

```diff
@@ -19,13 +23,15 @@ export default class TimePickerExampleComplex extends React.Component {
           ref="picker12hr"
           format="ampm"
           hintText="12hr Format"
-          onChange={this.handleChangeTimePicker24}
+          value={this.state.value12}
+          onChange={this.handleChangeTimePicker12}
         />
         <TimePicker
           ref="picker24hr"
           format="24hr"
           hintText="24hr Format"
-          onChange={this.handleChangeTimePicker12}
+          value={this.state.value24}
+          onChange={this.handleChangeTimePicker24}
```

Comment:
Something changes.
Master branch:
![capture d ecran 2016-02-01 a 18 46 50](https://cloud.githubusercontent.com/assets/3165635/12725695/31ee4dd2-c914-11e5-82a4-a8e40d6190c2.png)
This branch:
![capture d ecran 2016-02-01 a 18 46 17](https://cloud.githubusercontent.com/assets/3165635/12725682/223d77c8-c914-11e5-95f0-e5a5c47dde87.png)